lib/src/views/misc/interactive_grid_tile.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:rescado/src/views/buttons/floating_button.dart';
+
+class InteractiveGridTile extends StatefulWidget {
+  final String image;
+  final FloatingButton floatingButton;
```

Comment:
"Floating" button is not a very good name tbh. We should consider renaming this to something more accurate like "RoundedButton" or something alike.